src/api/ApiContext.js

```diff
@@ -0,0 +1,4 @@
+import React from "react";
```

Comment:
Probably good to have a comment here explaning what it is for. Should probably name this file `AlgoquantApiContext` as well.